test/python/onnx_importer/import_onnx_tool.runlit

```diff
@@ -0,0 +1,3 @@
+# RUN: %PYTHON -m torch_mlir.tools.import_onnx %S/LeakyReLU.onnx | FileCheck %s
```

Comment:
I think there's some typo here with `%S/%s`. Rest looks good.

Reply:
Believe it or not, no this is right :/

Capital 'S' refers to the directory that contains the file (lower case 's').

I have to look it up every time: https://llvm.org/docs/CommandGuide/lit.html#substitutions

Reply:
Thanks for the info. I think I need to learn these macros.